tests/test_plot/test_matplotlib_import.py

```diff
@@ -36,7 +37,13 @@ def test_that_mpl_dynamically_imports():
     baseplot = BasePlot()
 
     assert "matplotlib" in sys.modules
-    assert "matplotlib.pyplot" not in sys.modules
+
+    import matplotlib as mpl
+
+    if versionparse(mpl.__version__) < versionparse("3.6"):
```

Comment:
I don´t understand this

Reply:
I think this is what made the test fail in roxenv -- and it may also allow removing the pytest skip.

This test failed on the new CI with the RMS Python 3.8 with matplotlib setup because it uses matplotlib 3.3.x

https://github.com/equinor/xtgeo/blob/793a62d9da15ac3c483c4f78d953a7c620c9eae3/src/xtgeo/plot/baseplot.py#L12-L21

So in the RMS Python 3.8 environment, pyplot will end up imported whenever BasePlot (and by extension anything that derives from it -- which is everything else) is initialized. But not so in the RMS Python 3.11 environment

Reply:
OK, lets try